tests/benchmarks/test_work_stealing.py

```diff
@@ -0,0 +1,111 @@
+import time
+
+import dask.array as da
+import distributed
+import numpy as np
+import pytest
+from coiled.v2 import Cluster
+from dask import delayed, utils
+from distributed import Client, wait
+from tornado.ioloop import PeriodicCallback
+
+
+@pytest.mark.xfail(reason="https://github.com/dask/distributed/issues/6573")
+def test_trivial_workload_should_not_cause_work_stealing(small_client):
+    root = delayed(lambda n: "x" * n)(utils.parse_bytes("1MiB"), dask_key_name="root")
+    results = [delayed(lambda *args: None)(root, i) for i in range(10000)]
+    futs = small_client.compute(results)
+    wait(futs)
+    _ = [fut.result() for fut in futs]
+
+    def count_work_stealing_events(dask_scheduler):
+        return len(dask_scheduler.events["stealing"])
+
+    count = small_client.run_on_scheduler(count_work_stealing_events)
+    assert count == 0
```

Comment:
I think this is too strict. **If** we wanted to assert anything here we should rather assert that the data is distributed uniformly but we will not be able to suppress all stealing requests. This is also something we'd need to test in distributed itself.

I actually think we don't need to assert anything here. Wall time measurements will be sufficient and we will not need to xfail this test

Reply:
if it's too fast to get any meaningful signal we should remove it again. asserting on this level should be out of scope for coiled-runtime